gui/control_main.py

```diff
@@ -3820,9 +3875,19 @@ def pixelSelect(self, event):
                 correctedC2C_x = correctedC2C_x * scale_x
                 correctedC2C_y = correctedC2C_y * scale_y
                 self.md2.centring_click.put(f"{correctedC2C_x} {correctedC2C_y}")
+                #logger.info('waiting for motor rotation')
```

Comment:
remove commented-out text - if necessary, use git history to retrieve it, unless it is likely to be used soon